fontforge/cvexport.c

```diff
@@ -702,17 +702,24 @@ static void MakeExportName(char *buffer, int blen,char *format_spec,
 	    ++format_spec;
 	    ch = *format_spec++;
 	    if ((bend = buffer+40)>end ) bend = end;
-	    if ( ch=='n' ) {
+	    if ( ch=='n' || ch=='N' ) {
+            pt=sc->name;
+            if (ch=='N') {
+                char *ligature = GetLigature(sc);
+                if (ligature != NULL) {
+                    pt = ligature;
+                }
+            }
 #if defined( __CygWin ) || defined(__Mac)
 		/* Windows file systems are not case conscious */
 		/*  nor is the default mac filesystem */
-		for ( pt=sc->name; *pt!='\0' && buffer<bend; ) {
+		for ( ; *pt!='\0' && buffer<bend; ) {
 		    if ( isupper( *pt ))
 			*buffer++ = '$';
 		    *buffer++ = *pt++;
 		}
 #else
-		for ( pt=sc->name; *pt!='\0' && buffer<bend; )
+		for ( ; *pt!='\0' && buffer<bend; )
 		    *buffer++ = *pt++;
 #endif
```

Comment:
Freeing here is more of a pain because if there is no ligature you just write the glyph name, and you don't want to free *that* because there's still a pointer to it. So you could copy it also at the start but then you'd just have to free it right away if there is a ligature. So it's probably easiest to just do

```
if (pt!=sc->name)
    free(pt);
```
It's not safe to do `free(ligature);` here because if `ch!='N'` that won't be initialized/current. But you could do
```
if (ch=='N')
    free(ligature);
```
instead.